### DISCUSSION TITLE
A Dropdown inside a Modal breaks

### DISCUSSION THREAD

**Discussion by tilnea** [ariakit/ariakit#1403] [Questions] [answered]
Hi,

I have a dropdown inside a modal, but since I don't have much space, it does not look very pretty.
Is there a way I can make the Dropdown appear "over" the modal?

https://user-images.githubusercontent.com/3692320/169959146-e41f89a4-1fbf-47bc-8dc3-3ad724ba8d15.mov

**Comment by diegohaz** [comment, marked as answer]
Use the `portal` prop on the popover component.
